root/components/common-macros.tt

```diff
@@ -803,7 +803,7 @@ END ~%]
 -%]
     [%- React.embed(c, 'static/scripts/common/components/WikipediaExtract', {
         entity => {entityType => entity.entity_type, gid => entity.gid},
-        wikipediaExtract => wikipedia_extract,
+        cachedWikipediaExtract => wikipedia_extract,
```

Comment:
This should help make clear what the prop's for, and that it won't affect the displayed extract if the prop changes. (The prop couldn't change here anyway.)